src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random2 = new Random();

```

Comment:
The variable `random2` should follow a more descriptive naming convention. Consider renaming it to `random` or `randomGenerator` for clarity.